nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/test/java/org/apache/nifi/processors/aws/s3/ITPutS3Object.java

```diff
@@ -88,6 +100,74 @@ public static void setupClass() {
         kmsKeyId = getKMSKey();
     }
 
+    @AfterEach
+    public void tearDown() {
+        // clear bucket content
+        AmazonS3 client = getClient();
+        List<S3ObjectSummary> objectSummaries = client.listObjects(BUCKET_NAME).getObjectSummaries();
+        if (!objectSummaries.isEmpty()) {
+            client.deleteObjects(new DeleteObjectsRequest(BUCKET_NAME)
+                    .withKeys(objectSummaries.stream()
+                            .map(S3ObjectSummary::getKey)
+                            .toArray(String[]::new)));
+        }
```

Comment:
[AbstractS3IT](https://github.com/apache/nifi/blob/49702f1f510cc33c9cb52bb5b9580009a1d8afe8/nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/test/java/org/apache/nifi/processors/aws/s3/AbstractS3IT.java#L133-L149) contains similar functionality. To avoid duplication, that logic could be extracted to `emptyBucket()` method which could be called here too.

Reply:
Fixed in [7b26f3b](https://github.com/apache/nifi/pull/8295/commits/7b26f3b4fd6641388eebf4ac0d59f17aa92a6a0e)